README.md

```diff
@@ -31,7 +31,8 @@ dest/ <- Contains compiled code. Don't edit anything in this folder!
 src/
 ├── index.pug <- Template for demo page (Creates: dest/index.html)
 └── scss/
-    ├── custom/ <- SCSS modules for non-Bootstrap components.
+    ├── custom/ <- SCSS for non-Bootstrap modules.
+    |     └── components/ <- SCSS for custom components.
```

Comment:
Just wanna double - does it make sense to have the custom components dir live here? @gvn @alanmoo